docs/release_process.md

```diff
@@ -2,16 +2,42 @@
 
 The release process for AWX is completely automated as of version 19.5.0.
 
+## Get latest release version and list of new work
+
+1. Get latest version of [AWX](https://github.com/ansible/awx/releases)
+2. Get latest version of [AWX Operator](https://github.com/ansible/awx-operator/releases).
+3. [Compare](https://github.com/ansible/awx/compare) `latest` against `devel`
+
+![Comparing Branches](img/compare_release_branch.png)
+
+The page will automatically update with a list of PRs that are in `AWX/devel` but not in `latest`.
+
+![PR Compare List](img/pr_compare_list.png)
+
+Use this list of PRs to decide if this is a Y-stream (major) release, or a Z-stream release (minor). Indicators of a Z-stream release:
```

Comment:
It seems like we use semver.
```suggestion
Use this list of PRs to decide if this is a Y-stream (minor) release, or a Z-stream release (patch). Indicators of a Z-stream release:
```
X.Y.Z - 

Given a version number MAJOR.MINOR.PATCH, increment the:

    MAJOR version when you make incompatible API changes,
    MINOR version when you add functionality in a backwards compatible manner, and
    PATCH version when you make backwards compatible bug fixes.

See: https://semver.org/#summary